examples/whisper/user_script.py

```diff
@@ -0,0 +1,28 @@
+from onnxruntime.transformers.models.whisper.whisper_decoder import WhisperDecoderInputs
+from onnxruntime.transformers.models.whisper.whisper_encoder_decoder_init import WhisperEncoderDecoderInitInputs
+
+
+def encoder_decoder_init_dummy_inputs(model):
+    model = model.load_model()
```

Comment:
Could we move this logic inside PyTorchModel's method to get dummy inputs?

Reply:
This is specific for the whisper model so it is not possible to add it to PyTorch model.

Reply:
In the future, when we have automatic dummy input generation for hf models, this won't be necessary anymore. But that is a big feature implementation and cannot be done in time.